jdhzzz-ct-usa.md

```diff
@@ -0,0 +1,5 @@
+# CT Near Hartford
+
+## Places to get beer
+- Corner Pug in West Hartford
+- Eli Cannon's in Middletown
```

Comment:
It's always good practice to have a blank line at the end of file. Tony Kerz would not merge PRs without that :)